packages/terra-modal-manager/tests/jest/ModalManager.test.jsx

```diff
@@ -31,257 +17,30 @@ describe('ModalManger', () => {
     expect(result).toMatchSnapshot();
   });
 
-  describe('- sizes -', () => {
-    it('should render the ModalManager with tiny size', () => {
-      const modalManager = (
-        <ModalManager
-          size="tiny"
-          {...requiredProps}
-        >
-          <TestContainer />
-        </ModalManager>
-      );
-
-      const result = shallow(modalManager);
-      expect(result).toMatchSnapshot();
-    });
-
-    it('should render the ModalManager with small size', () => {
```

Comment:
With all these test removals, are these cases being tested elsewhere?

Reply:
A lot of these are removed because the DisclosureManager owns most of this logic now, and it is now tested in that package. Also, with the transition away from Redux and a `connect`-ed component, there are a lot less props than there used to be on this component so less Jest tests are required. The nightwatch tests are also still there mostly untouched (and managed to find a bug in my new implementation, so +1 for those).

Reply:
Ok that makes sense. Thanks for the clarification.